challenges/objects-arrays.js

```diff
@@ -59,12 +91,25 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
-console.log(contactInfo);
+
+for(let i = 0; i < graduates.length; i++) {
+  contactInfo.push(graduates[i].first_name, graduates[i].email);
+}
+
+// console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
-console.log(uni);
+
+for(let i = 0; i < graduates.length; i++) {
+  if(graduates[i].university === "uni") {
+    uni.push(graduates[i].university);
```

Comment:
you would have to look to see if the university property contains the string "Uni"....there is a method in JS called includes that would help with this. Right now it is doing a check to see if there is a string "Uni" by itself anywhere in that property...so it will return false for everything.